catalogs/catalog-lakehouse-iceberg/src/main/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/IcebergRESTService.java

```diff
@@ -43,18 +43,22 @@ private void initServer(IcebergConfig icebergConfig) {
     MetricsSystem metricsSystem = GravitinoEnv.getInstance().metricsSystem();
     server.initialize(serverConfig, SERVICE_NAME, false /* shouldEnableUI */);
 
-    ResourceConfig config = new ResourceConfig();
-    config.packages("com.datastrato.gravitino.catalog.lakehouse.iceberg.web.rest");
+    ResourceConfig resourceConfig = new ResourceConfig();
+    resourceConfig.packages("com.datastrato.gravitino.catalog.lakehouse.iceberg.web.rest");
 
-    config.register(IcebergObjectMapperProvider.class).register(JacksonFeature.class);
-    config.register(IcebergExceptionMapper.class);
+    resourceConfig.register(IcebergObjectMapperProvider.class).register(JacksonFeature.class);
+    resourceConfig.register(IcebergExceptionMapper.class);
     HttpServerMetricsSource httpServerMetricsSource =
-        new HttpServerMetricsSource(MetricsSource.ICEBERG_REST_SERVER_METRIC_NAME, config, server);
+        new HttpServerMetricsSource(
+            MetricsSource.ICEBERG_REST_SERVER_METRIC_NAME,
+            resourceConfig,
+            server,
+            metricsSystem.getServerConfig());
```

Comment:
One thing I was thinking is that we don't have to change the interface a lot, using `GravitinoEnv.config()` can get the global configuration, do you think it is a better solution?

Reply:
if using `GravitinoEnv.config()`, seems no need to pass `config` to `MetricsSource` constructor too, WDYT?

Reply:
Yes, we can roll back everything.